unstructured/partition/html.py

```diff
@@ -123,6 +123,7 @@ def partition_html(
         document,
         include_page_breaks=include_page_breaks,
         last_modification_date=metadata_last_modified or last_modification_date,
+        **kwargs,
```

Comment:
In this file it looks like the only actual change is adding `**kwargs`. Could you clean up the rest of the diffs in here?

Reply:
Looking into this now